pkg/code-generator/cmd/main.go

```diff
@@ -97,9 +97,11 @@ func Generate(opts GenerateOptions) error {
 		if !compileProtos {
 			return false
 		}
-		for _, proj := range projectConfigs {
-			if strings.HasPrefix(protoFile, filepath.Dir(proj.ProjectFile)) {
-				return true
+		for _, skp := range soloKitProjects {
+			for _, vc := range skp.ApiGroup.VersionConfigs {
+				if strings.HasPrefix(protoFile, filepath.Dir(skp.ProjectFile)+"/"+vc.Version) {
```

Comment:
this means `solo-kit.json` is now expected to live (always) in the parent directory of the version dirs? if so, it might be useful to provide a log line warning or error if the user somehow misses this point